DNN Platform/Modules/Groups/Create.ascx

```diff
@@ -100,6 +100,12 @@
             $('#trMem').hide();
 
         });
+        $('.wizardStepBody').find('.dnnPrimaryAction').click(function () {
+            var btn = $(this);
+            if ($('.wizardStepBody').find('input[type="text"]').val().length > 0) {
```

Comment:
This will work if the group name is blank but it will disable even if the group name is invalid.